nipype/pipeline/plugins/legacymultiproc.py

```diff
@@ -184,6 +184,8 @@ def __init__(self, plugin_args=None):
         non_daemon = self.plugin_args.get('non_daemon', True)
         maxtasks = self.plugin_args.get('maxtasksperchild', 10)
         self.processors = self.plugin_args.get('n_procs', cpu_count())
+        if self.plugin_args.get('max_jobs', np.inf) < self.processors:
+            logger.warning("To limit processes, use n_procs")
```

Comment:
Also, shouldn't passing `'max_jobs'` be an error, not a warning? 

```suggestion
            raise ValueError("Unknown plugin argument 'max_jobs' - "
                             "to limit processes, use 'n_procs'")
```

Reply:
I'm fine with raising an error.